website/docs/overview.md

```diff
@@ -6,167 +6,63 @@ toc: true
- - There are two ways of querying the same underlying table: Read Optimized query and Snapshot query, depending on whether we chose query performance or freshness of data.
- - The semantics around when data from a commit is available to a query changes in a subtle way for a read optimized query. Note, that such a query
- running at 10:10, wont see data after 10:05 above, while a snapshot query always sees the freshest data.
- - When we trigger compaction & what it decides to compact hold all the key to solving these hard problems. By implementing a compacting
- strategy, where we aggressively compact the latest partitions compared to older partitions, we could ensure the read optimized queries see data
- published within X minutes in a consistent fashion.
-
-The intention of merge on read table is to enable near real-time processing directly on top of DFS, as opposed to copying
-data out to specialized systems, which may not be able to handle the data volume. There are also a few secondary side benefits to 
-this table such as reduced write amplification by avoiding synchronous merge of data, i.e, the amount of data written per 1 bytes of data in a batch
-
-
+Welcome to Apache Hudi! This overview will provide a high level summary of what Apache Hudi is and will orient you on 
+how to learn more to get started.
+
+## What is Apache Hudi
+Apache Hudi (pronounced “hoodie”) is the next generation [streaming data lake platform](/blog/2021/07/21/streaming-data-lake-platform). 
+Apache Hudi brings core warehouse and database functionality directly to a data lake. Hudi provides [tables](/docs/next/table_management), 
+[transactions](/docs/next/timeline), [efficient upserts/deletes](/docs/next/write_operations), [advanced indexes](/docs/next/indexing), 
+[streaming ingestion services](/docs/next/hoodie_deltastreamer), data [clustering](/docs/next/clustering)/[compaction](/docs/next/compaction) optimizations, 
+and [concurrency](/docs/next/concurrency_control) all while keeping your data in open source file formats.
+
+Not only is Apache Hudi great for streaming workloads, but it also allows you to create efficient incremental batch pipelines. 
+Read the docs for more [use case descriptions](/docs/use_cases) and check out [who's using Hudi](/powered-by), to see how some of the 
+largest data lakes in the world including [Uber](https://eng.uber.com/uber-big-data-platform/), [Amazon](https://aws.amazon.com/blogs/big-data/how-amazon-transportation-service-enabled-near-real-time-event-analytics-at-petabyte-scale-using-aws-glue-with-apache-hudi/),
+[ByteDance](http://hudi.apache.org/blog/2021/09/01/building-eb-level-data-lake-using-hudi-at-bytedance), 
+[Robinhood](https://s.apache.org/hudi-robinhood-talk) and more are transforming their production data lakes with Hudi.
+
+Apache Hudi can easily be used on any [cloud storage platform](/docs/cloud), S3, ADLS, GCS, etc. 
```

Comment:
Does it make sense to either list all cloud stores or removed explicitly naming them since the link that you pointed includes all of them anyways ?

Reply:
updated